Configuration/PyReleaseValidation/python/MatrixReader.py

```diff
@@ -209,6 +209,9 @@ def readMatrix(self, fileNameIn, useInput=None, refRel=None, fromScratch=None):
             wfName = wfInfo[0]
             stepList = wfInfo[1]
             stepOverrides=wfInfo.overrides
+            #force use of RNTuple
+            stepOverrides['--rntuple_out']=''
```

Comment:
@smuzaffar We want to pass the `--rntuple_out` to `runTheMatrix.py` for the IB and PR tests of the RNTUPLE_X IB. Is it more convenient to do it in the `runTheMatrix` code (in the RNTUPLE_X branch) as here, or via the cms-bot?

Reply:
For this PR, that is not needed as he above line adds `--rntuple_out` to ALL workflows. In fact, I didn't add an option to `runTheMatrix.py` to use such a command line option.

Reply:
> May be add it conditionally e.g `if 'RNTUPLE_X' in os.environ['CMSSW_VERSION']` then add it?

Some kind of conditional will be needed if this logic would enter the `master` branch (or "eventually" and "when"). This PR targets the `RNTUPLE_X` branch, so _technically_ unconditional would be ok. Generally I'd like to keep the `runTheMatrix` and `cmsDriver` code as close as possible between `master` and `RNTUPLE_X`.

Reply:
I agree that keeping them close is important. That is why that change is in its own commit.